src/modules/reading-room/components/FilterMenu/FilterMenuMobile/FilterMenuMobile.tsx

```diff
@@ -99,9 +101,12 @@ const FilterMenuMobile: FC<FilterMenuMobileProps> = ({
 						</h4>
 
 						<TagList
+							className={clsx(styles['c-filter-menu-mobile__tags'], 'u-mb-0')}
 							closeIcon={<Icon className="u-text-left" name="times" />}
-							onTagClosed={() => null}
-							tags={[]}
+							onTagClosed={(id) =>
+								onRemoveValue?.(tags.filter((tag) => tag.id !== id))
```

Comment:
`id` (`key`) is not unique, this won't work with advanced filters.
Because of the way `react-select` works, `value` **is** unique, so you can use that to filter instead.